README.md

```diff
@@ -1,2 +1,68 @@
-# Covid
-A world wide covid stars
+# Covid stats
+
+> #### A web application for a company whcih provides commercial and scientific space travel services. In this application users can book rockets and join the space missions useing live data from the spaceX API.
+
+## Built With
+
+- React.js
+- Redux (@redux/toolkit)
+- React-Bootstrap
+
+## Additional Used
+
+- Api
+- KanbanBoard
+- Linters
+- Gitflow
+- npm
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`git clone git@github.com:miftah1991/space-travelers.git `
+
+Then `cd space-travelers`
```

Comment:
- [ ] Please change the link of the repo to the link of the covid repository.
- [ ] The same thing with `cd space-travlers`, try to change it to the name of the current project.